src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeAmpersandEmDashesFormatter.java

```diff
@@ -0,0 +1,31 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import org.jabref.logic.cleanup.Formatter;
+import org.jabref.logic.l10n.Localization;
+
+public class NormalizeAmpersandEmDashesFormatter extends Formatter {
+    @Override
+    public String getName() {
+        return Localization.lang("Normalize an ampersand and em dash");
+    }
+
+    @Override
+    public String getKey() {
+        return "normalize_ampersand_em_dash";
+    }
+
+    @Override
+    public String format(String value) {
+        return value.replace("&amp;#x2014;", "—");
```

Comment:
This is Unicode or plain ascii? I think, we fetch in plain BibTeX, thus, it should be --, shouldn't it?

Reply:
The source code is in Unicode. 

In `NormalizeEnDashesFormatter` en dash was replaced with `--`, em dash should be replaced with it too or something longer?

Reply:
Let's keep `--`.

Normally, it would be `---`, but this is very strange in Paper titles. Never saw it. Therefore, I proposed `--`.